extensions/expression-language/Expression.cpp

```diff
@@ -633,35 +633,46 @@ Value expr_escapeCsv(const std::vector<Value> &args) {
 }
 
 Value expr_format(const std::vector<Value> &args) {
-  std::chrono::milliseconds dur(args[0].asUnsignedLong());
-  std::chrono::system_clock::time_point dt(dur);
-  auto zone = date::current_zone();
-  if (args.size() > 2) {
-    zone = date::locate_zone(args[2].asString());
-  }
-  auto t = date::make_zoned(zone, dt);
-  std::stringstream result_s;
-  result_s << date::format(args[1].asString(), t);
-  return Value(result_s.str());
+  using std::chrono::milliseconds;
+
+  date::sys_time<milliseconds> utc_time_point{milliseconds(args[0].asUnsignedLong())};
```

Comment:
Is this really UTC? I thought sys_time is using the local timezone.

Reply:
yes this uses [std::chrono::system_clock](https://en.cppreference.com/w/cpp/chrono/system_clock) which is a UTC clock, for localtime you would need date::local_time instead